<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preview3D</class>
 <widget class="QWidget" name="Preview3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>773</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="buttonBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbTop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbBottom">
        <property name="text">
         <string>Bottom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbLeft">
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbRight">
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbFront">
        <property name="text">
         <string>Front</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbFit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbWireframe">
        <property name="text">
         <string>Wire</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbHide">
        <property name="text">
         <string>Hide</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbBack">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbIso1">
        <property name="text">
         <string>Iso1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbIso2">
        <property name="text">
         <string>Iso2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbIso3">
        <property name="text">
         <string>Iso3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbIso4">
        <property name="text">
         <string>Iso4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="notebook">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <!--
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
     -->
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
